src/components/select/SingleSelectConnected.tsx

```diff
@@ -9,13 +9,15 @@ import {Content} from '../content/Content';
 import {IItemBoxProps} from '../itemBox/ItemBox';
 import {ISelectButtonProps, ISelectProps, SelectConnected} from './SelectConnected';
 import {SelectSelector} from './SelectSelector';
+import * as styles from './styles/SingleSelect.scss';
 
 export interface ISingleSelectOwnProps extends ISelectProps {
     placeholder?: string;
     toggleClasses?: string;
     onSelectOptionCallback?: (option: string) => void;
     items?: IItemBoxProps[];
     buttonPrepend?: React.ReactNode;
+    fixedWidth?: boolean;

```

Comment:
I think it would be clearer if we did the inverse logic: the optional prop would be called `unfixedWidth` or a synonym and would have to be set to true if we want the dropdown to have an unset width. This would remove the need of setting a defaultProp

Reply:
done !